test/e2e/util.go

```diff
@@ -115,3 +115,26 @@ func patchAndCheckConsoleCLIDownloads(t *testing.T, client *framework.ClientSet,
 	})
 	return err
 }
+
+func patchAndCheckPodDisruptionBudget(t *testing.T, client *framework.ClientSet, isOperatorManaged bool) error {
+	t.Logf("patching Selector on the console PodDisruptionBudget")
+	pdb, err := client.PodDisruptionBudget.PodDisruptionBudgets(consoleapi.OpenShiftConsoleNamespace).Patch(context.TODO(), consoleapi.OpenShiftConsoleRouteName, types.MergePatchType, []byte(`{"spec": {"port": {"targetPort": "http"}}}`), metav1.PatchOptions{})
```

Comment:
we should be patching fields that are present in the PDB kind. `spec.port.targetPort` is Routes specific